mozilla/blurts-server
# Add empty state for the breach list

# References: 
Jira: MNTOR-1086
Figma: https://www.figma.com/file/gvyyqS9egsGjsH0LUDqT1m/Breach-Resolution-Flow-Design?node-id=654%3A16313&t=I1S7G3TnYG1IGh0z-4



# Description

This adds an empty state for the list of breaches. The acceptance criteria also mentioned telemetry, but I assume that that is depending on @rhelmer's work?

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/217830926-740ddfdf-4c7b-4e96-ae29-8580d33b5039.png)

![Screen Shot 2023-02-09 at 15 05 49](https://user-images.githubusercontent.com/4251/217834712-bcd87c63-e148-44c5-8806-c5f004fbf29c.png)

![image](https://user-images.githubusercontent.com/4251/217831097-a1c01735-54a7-40b2-a37c-72144c0267f4.png)

![Screen Shot 2023-02-09 at 15 05 37](https://user-images.githubusercontent.com/4251/217834795-46dc8357-c172-4442-b58d-9b59702f88c5.png)

# How to test

1. Open the breaches page for an email address with zero breaches - it should show the "No breaches found" message and hide the "Unresolved / Resolved" tabs.
2. Open the breaches page for an email address with more than zero unresolved breaches - it should just list those breaches.
3. Open the breaches page for an email address with more than zero resolved, and no unesolved breaches - it should show the "All breaches resolved" under the "Unresolved" tab, and list all breaches under the "Resolved" tab.
4. 1 and 3 above should show an "Add email address" button if the user has not reached their email address quota. Once they do, it should be hidden.

# Checklist (Definition of Done)
- [ ] I've added a unit test to test for potential regressions of this bug. - Don't know how to - do we have unit tests for UI?
- [ ] All acceptance criteria are met. - No telemetry, strings haven't been localised yet, and a couple of open questions inline.
